README.md

```diff
@@ -168,10 +168,21 @@ Usage of /gcp-workload-identity-federation-webhook:
 ### Pre-requisites
 
 - cert-manager: See [cert-manager installation](https://cert-manager.io/docs/installation/)
-- (optional) prometheus-operator: See https://github.com/prometheus-operator/prometheus-operator
+- prometheus-operator: See https://github.com/prometheus-operator/prometheus-operator
```

Comment:
Without Prometheus Operator, the installation of this controller will fail. However, Prometheus Operator is not required for this controller to work, so optional is preferred.

```diff
diff --git a/README.md b/README.md
index 39a3e9a..f0a4d30 100644
--- a/README.md
+++ b/README.md
@@ -168,7 +168,7 @@ Usage of /gcp-workload-identity-federation-webhook:
 ### Pre-requisites

 - cert-manager: See [cert-manager installation](https://cert-manager.io/docs/installation/)
-- prometheus-operator: See https://github.com/prometheus-operator/prometheus-operator
+- (optional) prometheus-operator: See https://github.com/prometheus-operator/prometheus-operator

 ### Deploy

diff --git a/charts/gcp-workload-identity-federation-webhook/templates/controller-manager-metrics-monitor.yaml b/charts/gcp-workload-identity-federation-webhook/templates/controller-manager-metrics-monitor.yaml
index 001654a..d9917eb 100644
--- a/charts/gcp-workload-identity-federation-webhook/templates/controller-manager-metrics-monitor.yaml
+++ b/charts/gcp-workload-identity-federation-webhook/templates/controller-manager-metrics-monitor.yaml
@@ -1,3 +1,4 @@
+{{- if .Values.serviceMonitor.enabled }}
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
@@ -17,3 +18,4 @@ spec:
     matchLabels:
       control-plane: controller-manager
     {{- include "gcp-workload-identity-federation-webhook.selectorLabels" . | nindent 6 }}
+{{- end }}
diff --git a/charts/gcp-workload-identity-federation-webhook/values.yaml b/charts/gcp-workload-identity-federation-webhook/values.yaml
index c09c3c8..1740ada 100644
--- a/charts/gcp-workload-identity-federation-webhook/values.yaml
+++ b/charts/gcp-workload-identity-federation-webhook/values.yaml
@@ -49,6 +49,9 @@ controllerManager:

 kubernetesClusterDomain: cluster.local

+serviceMonitor:
+  enabled: false
+
 metricsService:
   ports:
   - name: metrics
```

Reply:
👍